docs/proposals/20200304-autoscaler-scale-ux.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Cluster Autoscaler Scale from Zero UX
+authors:
+  - "@michaelgugino"
+reviewers:
+  - "@detiber"
+creation-date: 2020-03-04
+creation-date: 2020-03-04
+status:
+---
+
+# Cluster Autoscaler Scale from Zero UX
+
+## Summary
+Cluster Autoscaler supports scale from zero for cloud-native providers. This is done via compiling CPU, GPU, and RAM info into the cluster autoscaler at build time for each supported cloud.
```

Comment:
>This is done via compiling CPU, GPU, and RAM info into the cluster autoscaler at build time for each supported cloud.

I'd say this is actually a  provider implementation choice on how they decide to implement the `TemplateNodeInfo` interface, e.g gce does it dynamically https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/cloudprovider/gce/gce_cloud_provider.go#L332
https://github.com/kubernetes/autoscaler/blob/589cd2af9c25088ce3408eeb88ba9239201d005d/cluster-autoscaler/cloudprovider/gce/gce_manager.go#L491-L504